addons/account/models/chart_template.py

```diff
@@ -1154,8 +1154,15 @@ def mapping_getter(*args):
                 else:
                     format_tag = re.sub(r'\s+', ' ', tag.strip())
                     mapped_tag = tags.get(format_tag)
-                    if not mapped_tag:
-                        raise UserError(self.env._('Error while loading the localization. You should probably update your localization app first.'))
+                    country = self.env['res.country'].browse(country_id)
+                    message = self.env._(
+                        'Error while loading the localization: missing tax tag %(tag_name)s for country %(country_name)s. You should probably update your localization app first.',
+                        tag_name=format_tag, country_name=country.name)
+                    if not self._context.get('ignore_missing_tags'):
+                        raise UserError(message)
+                    else:
+                        _logger.error(message)
+                        continue
```

Comment:
```suggestion
                    if not mapped_tag:
                      country = self.env['res.country'].browse(country_id)
                      message = self.env._(
                          'Error while loading the localization: missing tax tag %(tag_name)s for country %(country_name)s. You should probably update your localization app first.',
                          tag_name=format_tag, country_name=country.name,
                      )
                      if not self._context.get('ignore_missing_tags'):
                          raise UserError(message)
                      else:
                          _logger.error(message)
                          continue
```
Small mistake here, it's going to raise every time.
I would also browse outside of the loop.

Thank you